tests/longevity-encryption-at-rest-200GB-6h.yaml

```diff
@@ -27,6 +27,22 @@ authorizer: 'CassandraAuthorizer'
 pre_create_schema: True
 scylla_encryption_options: "{ 'cipher_algorithm' : 'AES/ECB/PKCS5Padding', 'secret_key_strength' : 128, 'key_provider': 'LocalFileSystemKeyProviderFactory', 'secret_key_file': '/tmp/secret_key'}"
 
+# enable system_info_encryption, config kmip_hosts
```

Comment:
why do you keep maintaining old YAMLs?

Reply:
I didn't use the old YAMLs.

Can we rename `tests/` to `discard_tests/`, or `discard_tests_YAMLs/`?  Or just remove it.
/CC @roydahan @fruch